src/Components/Blazor/DevServer/src/Server/Program.cs

```diff
@@ -18,12 +25,24 @@ public class Program
         /// <summary>
         /// Intended for framework test use only.
         /// </summary>
-        public static IWebHost BuildWebHost(string[] args) =>
-            WebHost.CreateDefaultBuilder(args)
-                .UseConfiguration(new ConfigurationBuilder()
-                    .AddCommandLine(args)
-                    .Build())
-                .UseStartup<Startup>()
-                .Build();
+        public static IHost BuildWebHost(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureHostConfiguration(cb => {
+                    var applicationPath = args.SkipWhile(a => a != "--applicationpath").Skip(1).FirstOrDefault();
+                    var name = Path.ChangeExtension(applicationPath,".StaticWebAssets.xml");
+
+                    if (name != null)
```

Comment:
Nit: I don't think `name` can be null here anyway, because if `applicationPath` was null there would already have been an exception on L32.

Reply:
Also, just to check, is there any chance we need to be checking whether `File.Exists(name)` before proceeding with the rest of the static assets code paths below? I'm assuming it's possible for no such manifest file to exist.

Reply:
The extensions in WebHost already check if the file exist, and noop otherwise. I wouldn't want to do anything catastrophic here like throwing an exception as I don't want to make a bug/change in this logic to make the server unusable.

We have an E2E test for it, so we'll know if things change.